src/graph/test/GoTest.cpp

```diff
@@ -380,7 +380,90 @@ TEST_F(GoTest, MULTI_EDGES) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
-
+    {
+        cpp2::ExecutionResponse resp;
```

Comment:
Could you add a test to cover reversely multi-edge and the result has different props on the different edges?